src/segment-loader.js

```diff
@@ -672,27 +671,17 @@ export default class SegmentLoader extends videojs.EventTarget {
       return null;
     }
 
-    this.logger_('checkBuffer_',
```

Comment:
Any specific reason for removing this and other logs that currently exist?

Reply:
@mjneil was removing some excessive logging